src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,141 @@
+package duke.storage;
+
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import duke.commands.Command;
+import duke.parser.Parser;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.ToDo;
+import duke.tasklist.TaskList;
+
+/**
+ * Class the simulates the storing and retrieving of information into the hard-disk
+ */
+public class Storage {
+    // might use it at the end
```

Comment:
I like the use of ellipsis to simulate loading duration.

Reply:
Thanks